app/views/partials/_logged_in_current_user.erb

```diff
@@ -0,0 +1,7 @@
+<div>
+    <p> Suggestions <%= user.suggestions.length %></p>
+    <p> Up Votes Given <%= user.up_votes.length %></p>
+    <p> Users Supported <%= user.supports.to_a.uniq.length %></p>
+    <p> Up Votes Recieved <%= user.suggestions_up_voted.length %></p>
+    <p> Supporters <%= user.suggestions_up_voted.to_a.uniq.length %></p>
```

Comment:
Call `count` instead of `length` so these calculations are performed by SQL queries, which are much faster.